jvb/src/main/java/org/jitsi/videobridge/Endpoint.java

```diff
@@ -1123,13 +1128,18 @@ public void requestKeyframe()
      */
     private void setMediaSources(MediaSourceDesc[] mediaSources)
     {
+        boolean wasEmpty = transceiver.getMediaSources().length == 0;
         if (transceiver.setMediaSources(mediaSources))
         {
             eventEmitter.fireEventSync(handler -> {
                 handler.sourcesChanged();
                 return Unit.INSTANCE;
             });
         }
+        if (wasEmpty)
```

Comment:
Does this need to execute when there was no change to the sources (i.e. shoul it be under the `if` above)|?

Reply:
If the media sources is still empty, this will be a no-op when it hits the empty media source list in `sendVideoConstraints`, but I could move it inside if you think it'd be cleaner.